tests/test_strategy.py

```diff
@@ -112,9 +113,9 @@ def test_warmstart(self):
             self.strat.add_data(np.r_[1.0, 1.0], [1])
 
         self.assertEqual(
-            self.strat.model.fit.call_count, 4
+            self.strat.model.fit.call_count, 5
```

Comment:
What has this changed?

Reply:
Because now we are fitting the first time checking for `_is_first_fit` in Strategy.